src/documentation/pages/components/footer/examples/_external-lumen-portal.vue

```diff
@@ -0,0 +1,530 @@
+<template lang="pug">
+<ComponentExample title="External" id="external-lumen-portal" :tabs="exampleTabs">
+  p.-text(slot='example-description')
+    | Show the external footer for all public facing webpages.
+  <Wrapper slot='example'>
+    footer.chi-footer
```

Comment:
Check this in every example.

```suggestion
    footer.chi-footer(slot='example')
```